src/documentcolorcollection.js

```diff
@@ -23,32 +23,43 @@ export default class DocumentColorCollection extends Collection {
 		 * @readonly
 		 * @member {Boolean} #isEmpty
 		 */
-		this.set( 'isEmpty', true );
+		this.set( 'isEmpty', !this.length );
 	}
 
 	/**
-	 * Adds a color to the document color collection.
+	 * Adds a color (or colors) to the document color collection.
 	 *
 	 * This method ensures that no color duplicates are inserted (compared using
 	 * the color value of the {@link module:ui/colorgrid/colorgrid~ColorDefinition}).
 	 *
 	 * If the item does not have an ID, it will be automatically generated and set on the item.
 	 *
 	 * @chainable
-	 * @param {module:ui/colorgrid/colorgrid~ColorDefinition} item
-	 * @param {Number} [index] The position of the item in the collection. The item
-	 * is pushed to the collection when `index` is not specified.
+	 * @param {...(module:ui/colorgrid/colorgrid~ColorDefinition)} items
```

Comment:
I was looking at a similar construct in the uitls PR and AFAIC the `()` is not needed for at least primitives. Maybe in this case we could also drop them?